ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Expected Behavior
+
+## Actual Behavior
+
+## Steps to Reproduce Behavior
+
+## Issue Severity Classification - Select from:
+### (1) Severe   - Fully impedes usage of `ProjectTemplate`. This condition requires an immediate solution.  `ProjectTemplate` is unable to operate or caused other critical software to fail and there is no acceptable way to work around the problem.
+### (2) Moderate - Significant impact.  `ProjectTemplate` is usable but is limited. Work around exists, but requires significant effort. [Incomplete/incorrect documentation, poor or unexplained log messages where no clear error was evident, side effect is observed which does not significantly harm operations etc]
+### (3) Low      - Minimal impact. Issue is an annoyance, but I can get work done. This indicates the problem causes little impact on operations or that a reasonable circumvention to the problem has been implemented. [Usability quirks, requires minor documentation updates, or could be enhance with some minor changes to the function.]
```

Comment:
I'm missing an option for feature requests. Those are actually pretty common on this project, and I disagree with https://github.com/maqetta/maqetta/wiki/Severity-and-Priority-of-Issues that `enhancement` shouldn't be reported this way. Of course a nice PR is preferable, but we can't expect everyone to be able to actually contribute code themselves.

Reply:
Yea, we should get something for feature requests too. I'll split the template into "bugs/issues" and "feature requests".